src/main/java/parkSystem/park/common/dto/response/CommonResponse.java

```diff
@@ -0,0 +1,11 @@
+package parkSystem.park.common.dto.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class CommonResponse {
```

Comment:
dto를 사용할떄 record를 사용해보는것도 좋은 방법 같습니다!!!
```suggestion
public class CommonResponse {
```